gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/test/java/io/gravitee/rest/api/management/v2/rest/spring/ResourceContextConfiguration.java

```diff
@@ -504,4 +509,29 @@ public UndeploySharedPolicyGroupUseCase undeploySharedPolicyGroupUseCase() {
     public SearchSharedPolicyGroupHistoryUseCase searchSharedPolicyGroupHistoryUseCase() {
         return mock(SearchSharedPolicyGroupHistoryUseCase.class);
     }
+
```

Comment:
I don't believe that the use cases should be mocked. Ideally, the use case is executed normally and it's the InMemory repositories are used by default for the use cases. This allows for almost true end-to-end testing with the Resource tests.